homeassistant/components/amcrest/binary_sensor.py

```diff
@@ -111,6 +115,12 @@ class AmcrestSensorEntityDescription(BinarySensorEntityDescription):
         device_class=BinarySensorDeviceClass.CONNECTIVITY,
         should_poll=True,
     ),
+    AmcrestSensorEntityDescription(
+        key=_DOORBELL_KEY,
+        name=_DOORBELL_NAME,
+        device_class=BinarySensorDeviceClass.MOTION,
```

Comment:
I think occupancy or presence would be more correct than motion here

Reply:
There's no perfectly matching device class for a doorbell button, but `motion` seems like the closest fit to me. It's something that's also typically used as an alert and it uses the value labels `detected`/`clear`.  Those seem much more appropriate than `home`/`away` or `occupied`/`unoccupied`.

If using `motion` here is inappropriate for some reason, it may be better not to use a device class at all. 🤷

Reply:
Hmm... it looks like I actually misread the documentation.  It seems the `occupancy` device class also uses `detected`/`clear`.  I'll switch it to that instead.  Thank you for catching that @Kane610! :)